hazelcast/src/main/java/com/hazelcast/monitor/impl/LocalMapStatsImpl.java

```diff
@@ -301,6 +313,73 @@ public void setNearCacheStats(NearCacheStats nearCacheStats) {
     }
 
     @Override
+    public void updateContainsStats(long startTime) {
+        incrementHits();
+        setLastAccessTime(startTime);
+    }
+
+    @Override
+    public void updateGetStats(long startTime, Object result) {
+        long now = getNow();
+        final long duration = now - startTime;
+        incrementGets(duration);
+
+        if (result != null) {
+            incrementHits();
+            setLastAccessTime(now);
+        }
+    }
+
+    @Override
+    public void updateGetStats(long startTime, long hits) {
+        incrementHits(hits);
+        setLastAccessTime(startTime);
+    }
+
+    @Override
+    public void updatePutStats(long startTime) {
```

Comment:
Seems better to use existing code here to decrease code duplications 
`@Override
    public void updatePutStats(long startTime) {
        updatePutStats(startTime, 1);
    }`